fermipy/defaults.py

```diff
@@ -101,7 +101,6 @@ def make_default_dict(d):
     'resample': (True, '', bool),
     'srcmap': (None, '', str),
     'bexpmap': (None, '', str),
-    'wmap': (None, 'Likelihood weights map.', str),
```

Comment:
Hi Matt-

You're right on both counts- emacs and a copied version. Is it possible for
the master to revert back to most recent version, then I can check it out,
remake my changes to defaults and gtanalysis (both minor) and push it back
to the dev branch?

Thanks and again, sorry for the mess

Andy

On Thu, Sep 1, 2016 at 12:43 AM, Matthew Wood <notifications@github.com>
wrote:

> In fermipy/defaults.py
> <https://github.com/fermiPy/fermipy/pull/80#discussion_r77113206>:
>
> > @@ -101,7 +101,6 @@ def make_default_dict(d):
> >      'resample': (True, '', bool),
> >      'srcmap': (None, '', str),
> >      'bexpmap': (None, '', str),
> > -    'wmap': (None, 'Likelihood weights map.', str),
>
> git should automatically handle merging changes in the dev and master
> branches.
>
> What seems to have happened in this case is that you made your changes
> against older versions of gtanalysis.py and defaults.py and then
> committed those back into the lightcurve-dev branch. I don't see how this
> could have happened unless you were editing the file somewhere else and
> then subsequently copied it back into the git repository. In general before
> making commits I would always run git diff to double-check that the
> changes you're committing are what you expect. Maybe your text editor was
> cacheing an older version of the file somehow? Are you using emacs?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fermiPy/fermipy/pull/80/files/439d4ebc8d1496b32bbe6993e70bf4149628eba7..ec075b26ee7244283d6c6c4e634325ad91329f2a#r77113206>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJllvdNmEVk_d1eDJWrL22v2UedXTBLpks5qllf2gaJpZM4JtMuE>
> .
>



-- 
Dr. Andrew W. Smith
Research Associate Professor
Department of Physics
University of Maryland, College Park
1117 John S. Toll Building #082
College Park, MD 20742